app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -35,6 +43,14 @@ public void setListeners() {
         loginButton.setOnClickListener(view ->
                 getPresenter().loginButtonClicked(username.getText().toString(), password.getText().toString())
         );
+        signupButton.setOnClickListener(view -> {
+            Intent intent = new Intent(getActivity(), SignupActivity.class);
```

Comment:
Remember that Fragment will not decide, so you have to call Presenter with `onSignUpButtonClicked()` and then presenter will tell that Fragment has to do an Intent